src/core/layout/scrollable-list/scrollable-list-buttons.js

```diff
@@ -32,16 +33,20 @@ const createGelButton = (scene, item) => {
         channel: gelConfig.channel,
         name: id,
     });
-
-    return fp.flow(scaleButton, overlays1Wide)({ scene, gelButton, config, item });
+    return fp.flow(scaleButton, makeAccessible, overlays1Wide)({ scene, gelButton, config, item });
 };
 
 const scaleButton = args => {
     const { scene, config, gelButton } = args;
-    const safeArea = scene.layout.getSafeArea();
+    const safeArea = scene.layout.getSafeArea({}, false);
     const scaleFactor = (safeArea.width - config.space * 4) / gelButton.width;
     gelButton.setScale(scaleFactor);
     return args;
 };
 
+const makeAccessible = args => {
+    accessibilify(args.gelButton);
+    return args;
```

Comment:
nitpick but I don't think this needs to return anything, does it?

Reply:
correct, & good spot- it did at one point but now the only reason to return args is to keep it in the `fp.flow` call above. Will refactor.